tajo-common/src/main/java/org/apache/tajo/conf/TajoConf.java

```diff
@@ -377,6 +381,7 @@ public static int setDateOrder(int dateOrder) {
       // ResultSet ---------------------------------------------------------
     $RESULT_SET_FETCH_ROWNUM("tajo.resultset.fetch.rownum", 200),
```

Comment:
When we use compression, ``200`` may be too small to get some performance benefits from compression. Later, we may need some experiments to find the best row number for compression or non-compression mode.